app/src/main/kotlin/com/saltedge/authenticator/model/db/ConnectionsRepository.kt

```diff
@@ -101,6 +102,20 @@ object ConnectionsRepository : ConnectionsRepositoryAbs {
         }
     }
 
+    /**
+     * Get Connection by connection url
```

Comment:
this function returns list of connections not single connection. fix description